TESTING.md

```diff
@@ -123,4 +123,23 @@ By creating a Pull Request the following pipelines will be triggered
 - [Buildkite `[Test] Prisma TypeScript`](https://buildkite.com/prisma/test-prisma-typescript)
 - [GitHub Action `CI`](https://github.com/prisma/prisma/blob/main/.github/workflows/test.yml)
 
-They are both running the same tests but with different Node.js version and will need to be successful before merging.
+They are both running the same tests but with different Node.js version and will need to be successful before merging ("flaky" tests might show up and might be ignored).
+
+### Publishing an integration version of all the packages
+
+If a branch name starts with `integration/` like `integration/fix-all-the-things` the [Buildkite `[Release] Prisma TypeScript`](https://buildkite.com/prisma/release-prisma-typescript) pipeline will be triggered.
+If tests pass a new version of the packages will be published like `3.12.0-fix-all-the-things.1` (where `3.12.0-` is the current dev version prefix, and `.1` the first version published from this integration branch)
+
+To make a PR which will release an integration version, the name of the branch of the PR would need to start with `integration/`.
+The `Buildkite [Release] Prisma TypeScript` will show its status in the PR checks and might take up to 30min to finish.
+
+Once published to npm the version will need to be installed with the exact version like:
+
+```
+npm install -D prisma@3.12.0-fix-all-the-things.1
+
+# or executed with npx like
+npx prisma@3.12.0-fix-all-the-things.1
+```
+
```

Comment:
(Note that Npm version upgrades or the update notifier in Prisma CLI might behave weird and unexpected with these integration versions.)